gradle.properties

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2021 Alliander N.V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Version Plugins and Dependencies
+jandexPluginVersion=0.9.0
+quarkusPluginVersion=1.13.3.Final
+
+quarkusPlatformVersion=1.13.3.Final
```

Comment:
We could use 1.13.4.Final for now (not necessary, but then we're using the latest)